tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1910,88 +1801,67 @@ def test_degraded_writes_till_kpods_fail(self):
+            self.pod_dict[pod_name] = pod_data
+            self.pod_name_list.append(pod_name)
             self.restore_pod = self.deploy = True
             self.restore_method = const.RESTORE_DEPLOYMENT_K8S
-            self.pod_dict[pod_name] = pod_data
-            LOGGER.info("Step 2: Deleted %s pod %s by deleting deployment (unsafe)",
-                        count, pod_name)
+            assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+            LOGGER.info("Deleted %s data pod %s by deleting deployment (unsafe)", count, pod_name)
 
-            LOGGER.info("Step 3: Check cluster status")
-            resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
-            assert_utils.assert_false(resp[0], resp)
-            LOGGER.info("Step 3: Cluster is in degraded state")
-
-            LOGGER.info("Step 4: Check services status that were running on pods which are "
-                        "deleted")
-            hostname = self.pod_dict.get(pod_name)[0]
-            resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=[pod_name], fail=True,
-                                                               hostname=hostname)
-            assert_utils.assert_true(resp[0], resp[1])
-            pod_list.remove(pod_name)
-            LOGGER.info("Step 4: Services of pods are in offline state")
-
-            LOGGER.info("Step 5: Check services status on remaining pods %s", pod_list)
-            resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=pod_list, fail=False)
-            LOGGER.debug("Response: %s", resp)
-            assert_utils.assert_true(resp[0], resp)
-            LOGGER.info("Step 5: Services of remaining pods are in online state")
-
-            LOGGER.info("Step 6: Perform WRITEs, READs and verify DI on the already created "
+            LOGGER.info("Step 3: Perform WRITEs, READs and verify DI on the already created "
                         "bucket")
             resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                         log_prefix=self.test_prefix,
-                                                        skipcleanup=True, nsamples=2, nclients=2)
+                                                        skipcleanup=True, nsamples=2, nclients=2,
+                                                        setup_s3bench=False)
             assert_utils.assert_true(resp[0], resp[1])
-            LOGGER.info("Step 6: Successfully performed WRITEs, READs and verify DI on the "
+            LOGGER.info("Step 3: Successfully performed WRITEs, READs and verify DI on the "
                         "written data")
 
-            LOGGER.info("Step 7: Perform WRITEs-READs-Verify with variable object sizes. 0B + ("
-                        "1KB - 512MB) on degraded cluster")
+            LOGGER.info("Step 4: Perform WRITEs-READs-Verify on new buckets with variable"
```

Comment:
Please add DTM flag condition.

Reply:
> Please add DTM flag condition.

We won't be able to test this one as, we required to create new buckets everytime one pod shutdown. I have already added skip marker there.